kt/entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/ext/TypeExtensions.kt

```diff
@@ -0,0 +1,176 @@
+package godot.entrygenerator.ext
```

Comment:
Same here, should be merged in a near future with what we have in api generator.